Classes.html

```diff
@@ -443,337 +455,327 @@ <h4>Declaration</h4>
 
-                        <a href="Classes/Venue.html" class="slightly-smaller">See more</a>
+                        <a href="Classes/TextOverlay.html" class="slightly-smaller">See more</a>
                       </div>
                       <div class="declaration">
                         <h4>Declaration</h4>
                         <div class="language">
                           <p class="aside-title">Swift</p>
-                          <pre class="highlight swift"><code><span class="kd">open</span> <span class="kd">class</span> <span class="kt">Venue</span></code></pre>
+                          <pre class="highlight swift"><code><span class="kd">public</span> <span class="kd">final</span> <span class="kd">class</span> <span class="kt">TextOverlay</span> <span class="p">:</span> <span class="kt"><a href="Classes/ImageOverlay.html">ImageOverlay</a></span></code></pre>
 
                         </div>
                       </div>
                     </section>
                   </div>
                 </li>
-              </ul>
-            </div>
-            <div class="task-group">
-              <ul>
                 <li class="item">
                   <div>
                     <code>
-                    <a name="/s:8Mappedin8CategoryC"></a>
-                    <a name="//apple_ref/swift/Class/Category" class="dashAnchor"></a>
-                    <a class="token" href="#/s:8Mappedin8CategoryC">Category</a>
+                    <a name="/s:8Mappedin10PathSolverC"></a>
+                    <a name="//apple_ref/swift/Class/PathSolver" class="dashAnchor"></a>
+                    <a class="token" href="#/s:8Mappedin10PathSolverC">PathSolver</a>
                     </code>
                   </div>
                   <div class="height-container">
                     <div class="pointer-container"></div>
                     <section class="section">
                       <div class="pointer"></div>
                       <div class="abstract">
-                        <p>A Category is a collection of similar Locations, which may have parent or
-child categories. For example, you may have a Fashion category, which has
-a Men&rsquo;s Fashion and Women&rsquo;s Fashion category. A Men&rsquo;s clothing store would
-belong to the Men’s Fashion category, but not necessarily be present in the
-Fashion category&rsquo;s list of Locations.</p>
+                        <p>The PathSolver is used to take a set of locations and create an ordered
+list of locations that is an efficent path though a venue.</p>
 
-                        <a href="Classes/Category.html" class="slightly-smaller">See more</a>
+<p>This solver cannot just give the best solution, as this a brute force
+solver would run in O(N!) time, which is not resonable if the number
+is bigger then half a dozen items.</p>
+
+<p>Instread of solving the problem all at once, the solver improves
+the solution continiouslly until a good enough solution is found.</p>
+
```

Comment:
A few typos here. 'Instread' and 'continiouslly'

Reply:
good catch